google-cloud/google-cloud.gemspec

```diff
@@ -44,4 +44,15 @@ Gem::Specification.new do |gem|
   gem.add_development_dependency "simplecov", "~> 0.9"
   gem.add_development_dependency "yard", "~> 0.9"
   gem.add_development_dependency "yard-doctest", "<= 0.1.8"
+
+  gem.post_install_message = <<-POSTINSTALL
+------------------------------
+Thank you for installing Google Cloud!
+
+IMPORTANT NOTICE:
+The google-cloud gem contains all the google-cloud-* gems.
+Instead of depending on this gem, we encourage you to install just
+the individual gems needed for your project.
```

Comment:
Is there something we can tell users about how to find what the "individual gems" are? (e.g. "search for google-cloud-<api-name>" or something)

Reply:
I agree this wording could be better. I would prefer something less passive. That said, I’m out of ideas on how to phrase it. Do you have any suggestions for what you would like to see?

Reply:
Oh never mind, I didn't even read the existing notice carefully enough. It does say what the individual gems are. Don't mind me. LGTM.